core/templates/services/svg-sanitizer.service.ts

```diff
@@ -71,21 +71,33 @@ export class SvgSanitizerService {
       // we need to replace these decimals with a letter so that it's easier
       // to process and validate the filenames.
       if (node.tagName.toLowerCase() === 'svg') {
-        dimensions.height = (
-          (node.getAttribute('height').match(/\d+\.*\d*/g)[0]).replace(
-            '.', 'd'));
-        dimensions.width = (
-          (node.getAttribute('width').match(/\d+\.*\d*/g)[0]).replace(
-            '.', 'd'));
+        let attrHeight = node.getAttribute('height');
+        if (attrHeight) {
+          let height = attrHeight.match(/\d+\.*\d*/g);
```

Comment:
It's odd to call a match object "height", "width", etc. Use a name that more semantically represents what it is describing, e.g. heightRegexpMatchResult or similar. Reserve "height", "width", etc. for the actual height, width, etc.

This comment applies throughout.

Reply:
done